rqt_gui_cpp/CMakeLists.txt

```diff
@@ -1,20 +1,40 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.5)
 project(rqt_gui_cpp)
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED COMPONENTS qt_gui qt_gui_cpp roscpp nodelet)
-catkin_package(
-	INCLUDE_DIRS include
-	LIBRARIES ${PROJECT_NAME}
-	CATKIN_DEPENDS qt_gui qt_gui_cpp roscpp nodelet
-)
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# Qt5 boilerplate options from http://doc.qt.io/qt-5/cmake-manual.html
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+set(CMAKE_AUTOMOC ON)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fPIC")
+find_package(ament_cmake REQUIRED)
+find_package(Qt5Widgets CONFIG REQUIRED)
+find_package(qt_gui REQUIRED)
+find_package(qt_gui_cpp REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(tinyxml_vendor REQUIRED)
```

Comment:
lol. Would that be a big change? Can you point me to an example migration from TinyXml -> TinyXml2

Reply:
It depends how extensively it is used, but probably not since only the plugin.xml and the package.xml have to be parsed. 

Check out the ros_pluginlib_plugin_provider.h here.
https://github.com/ros-visualization/qt_gui_core/compare/crystal-devel...ros2_port_windows?expand=1

Reply:
It doesn't look like tinyxml is even used in this package. I think I can delete it